client/src/components/Pages/JobsPage/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import axios from 'axios';
+
+import JobCard from '../../common/JobCard';
+
+class JobPage extends React.Component {
+  state = {
+    jobsData: [],
+  };
+
+  async componentDidMount() {
+    const response = await axios.get('/api/v1/jobs');
```

Comment:
What if there are no jobs for this user ??
Show the user notification for this case